lib/tessel/deploy.js

```diff
@@ -100,8 +108,10 @@ actions.execRemoteCommand = function(tessel, command, filepath) {
   });
 };
 
-actions.findProject = function(file) {
+actions.findProject = function(opts) {
   return new Promise(function(resolve, reject) {
+    var single = opts.single;
+    var file = opts.file;
```

Comment:
durr, sorry.  I need to write test for this, but could use help to get myself situated with the testing stubs and methodology we're using.